src/sentry/integrations/slack/unfurl/discover.py

```diff
@@ -44,6 +45,9 @@ def unfurl_discover(data, integration, links: List[UnfurlableUrl]) -> UnfurledUr
         params = link.args["query"]
         query_id = params.get("id", None)
 
+        user_id = params.get("user", None)
+        user = User.objects.get(id=user_id) if user_id else None
```

Comment:
Can we also verify that the user is in the same org.

You can use this filter

```
User.objects.get(
	id=user_id,
	sentry_orgmember_set__organization=org
)
```

Just want to make sure people don't try and do anything funky here :)